compute_endpoint/tests/unit/test_cli_behavior.py

```diff
@@ -1027,13 +1083,16 @@ def test_configure_ep_auth_policy_defaults(
     make_endpoint_dir,
     ep_name,
 ):
-    mock_login_manager = mocker.patch(f"{_MOCK_BASE}LoginManager")
+    mock_app = mock.Mock(spec=UserApp)
+    mocker.patch(f"{_MOCK_BASE}get_globus_app_with_scopes", return_value=mock_app)
+    mock_auth_client = mock.Mock(spec=ComputeAuthClient)
+    mocker.patch(f"{_MOCK_BASE}ComputeAuthClient", return_value=mock_auth_client)
     mock_create_auth_policy = mocker.patch(f"{_MOCK_BASE}create_auth_policy")
```

Comment:
This setup is used a couple of times; a couple of fixtures might be in order.

Reply:
Good call.